README.md

```diff
@@ -440,7 +441,7 @@ td.when(readFile('my-secret-doc.txt')).thenCallback(null, 'secrets!')
 Then, the subject might invoke readFile and pass an anonymous function:
 
 ```js
-readFile('my-secret-doc.txt', function (er, contents) {
+readFile('my-secret-doc.txt', function (err, contents) {
```

Comment:
I assume this was just a typo? `err` seems much more common as a JS param, but I could be wrong.